recipes/pycvodes/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pycvodes" %}
+{% set version = "0.10.5" %}
+{% set sha256 = "5c751e92661f1fd032856adda34667f6e8b3edf181222fed8d2070621c2e590c" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 200
+  skip: True  # [win]
+  features:
+    - blas_{{ variant }}  # [not win]
+
+requirements:
+  build:
+    - blas 1.1 {{ variant }}  # [not win]
+    - openblas 0.2.20*  # [not win]
+    - sundials >=3.1.0
+    - python
+    - setuptools
+    - pip
+    - numpy
```

Comment:
Is this using numpy C API?

Reply:
I believe so (I have Cython-function signatures such as: `cdef _reshape_roots(cnp.ndarray[cnp.float64_t, ndim=1] roots, int ny):`)

Reply:
See https://conda-forge.org/docs/meta.html#building-against-numpy